### PR TITLE
Add SQL config manager

### PR DESCRIPTION
**Pull request by mcm001**
Serializes settings using a sqlite database instead of just putting them on the filesystem. Ideally since sqlite deals with filesystem robustness stuff this should work a lot better

### CODE REVIEW

**Review by laviRZ — COMMENTED** [review]
There are also some TODO comments still left, idk if on purpose or by mistake

**Comment by laviRZ** [comment]
The "querry" typo is still present in multiple places

**Comment by mcm001** [comment]
Lol good catch, thanks!

**Review by Bankst — APPROVED** [review]

File: photon-client/package-lock.json
Comment (srimanachanta) [inline photon-client/package-lock.json:1]: There shouldn't be conflicts here

**Pull request merged**